src/Components/ItemDetails.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import dayjs from 'dayjs';
+import '../styles/Modal.css';
+
+export default function ItemDetails({ item }) {
+  const lastPurchaseDate = item.lastPurchased
+    ? dayjs(item.lastPurchased.toDate()).format('MMMM D, YYYY')
+    : '-';
```

Comment:
Maybe an EJ question... When LPD or NPD is blank and shows a "-", the screen reader reads "Next Purchase Date: dash". Should there be an aria label saying no info yet or will people know what dash means when hearing it?